src/core_plugins/kibana/public/index.scss

```diff
@@ -0,0 +1 @@
+@import '../../../../src/ui/public/styles/_styling_constants'
```

Comment:
I'd remove the prefixed underscore from `_styling_constants`

Reply:
And add a semicolon to the end

Reply:
And then in this file is where each core plugin would import the meta `index.scss` file?

Reply:
Traditionally (or maybe just me) underscores are used for files that don't compile themselves.

Reply:
This is the file that is our "Kibana invisibiles". If we made Kibana global_styling, it would probably live here (or be imported into it).

Then a plugin... say monitoring/index.scss, imports this file as a one liner knowing that anything it builds will have that scope.

Reply:
Example of the underscore stuff on the "partials" sections.

http://sass-lang.com/guide

Reply:
Oh shoot. I misread your comment. You meant the line itself. Doh!

Reply:
Haha, youp!